packages/preview/riesketcher/0.4.0/README.md

```diff
@@ -0,0 +1,15 @@
+# riesketcher
+
+A package to draw Riemann sums (and their plots) of a function with CeTZ.
+
+Usage example and docs: [manual.pdf](https://github.com/ThatOneCalculator/riesketcher/blob/main/manual.pdf)
```

Comment:
I first assumed that the functions would immediately draw the plot. Maybe you can add somewhere that the return value needs to be passed to `cetz.canvas`.